src/main/java/in/sakthi/service/ChannelService.java

```diff
@@ -0,0 +1,19 @@
+package in.sakthi.service;
+
+import java.util.HashMap;
+public class ChannelService {
+	
+	private static final HashMap<Integer, String> channels = new HashMap<Integer, String>();
+	static
+	{
+	channels.put(1,"TvShows");
+	channels.put(2,"Sports");
+	channels.put(3,"Movies");
+	channels.put(4, "News");
+	System.out.println("The channel List is: "+channels);
```

Comment:
Avoid hardcoding Channel Names. Fetch data from Database.
Check if this article is useful to write dao methods - https://github.com/nareshkumar-h/jdbc-dao-demo